model/postgres/postgres_resource.rb

```diff
@@ -29,12 +29,15 @@ def hyper_tag_name(project)
   def hostname
     if Config.postgres_service_hostname
       "#{server_name}.#{Config.postgres_service_hostname}"
-    else
+    elsif server.vm
       server.vm.ephemeral_net4.to_s
+    else
+      ""
```

Comment:
Why do we return an empty string instead of nil? Isn't nil more appropriate to indicate the absence of a value?